api/server/call_logs.go

```diff
@@ -63,5 +64,6 @@ func (s *Server) handleCallLogGet(c *gin.Context) {
 		}
 	}
 	// if we've reached this point it means that Fn didn't recognize Accepted content type
-	c.Status(http.StatusNotAcceptable)
+	WriteError(c, c.Writer, http.StatusNotAcceptable,
```

Comment:
every other call in the front end uses `handleErrorResponse`, with an error that implements `models.APIError`. please do the same here, so that we might log this in `handleErrorResponse`, add a metric for it, whatever else we may want to do in there so that we have telemetry around this and uniformity in the front end error handling.